bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala

```diff
@@ -0,0 +1,130 @@
+package edu.gemini.model.p1.overheads
+
+import java.time.Duration
+
+import edu.gemini.model.p1.immutable._
+
+import scalaz.Scalaz._
+
+// This is not the ideal package for this to live in, but to avoid circular bundle references, we put it here.
+sealed trait Overheads {
+  def partnerOverheadFraction: Double
+  def acquisitionOverhead: Duration
+  def typicalExpTime: Option[Duration]
+  def otherOverheadFraction: Double
+
+  // Careful here: if we convert to hours directly using java.time API, this will be a Long value representing
+  // the floor of the number of hours.
+  private def acqOverheadHrs = acquisitionOverhead.toMinutes.toDouble / 60.0
+
+  def calculate(intTime: TimeAmount): ObservationTimes = {
+    // Calculate everything in hours, as per REL-2985 formulae.
+    val progTimeHrs  = {
+      val visTimeHrs = Overheads.visTimeHrs
+      val intTimeHrs = intTime.toHours.value
+      val overheadTimeHrs = (1 + otherOverheadFraction) * intTimeHrs
+      val numAcqs = ((overheadTimeHrs / visTimeHrs).toInt + ((overheadTimeHrs % visTimeHrs > 0) ? 1 | 0)).toDouble
+
+      // Careful here: if we convert to hours directly using java.time API, this will be a Long value representing
+      // the floor of the number of hours.
+      val acqOverheadHrs = acquisitionOverhead.toMinutes.toDouble / 60.0
```

Comment:
The `acquisitionOverhead` could be type-changed from `Duration` to `Double` (or just `Long`) in mins, which is probably what I will do going forward since the use of `Duration` does seem to be a little unpredictable based on the fact that it uses `Long`s and takes the floor when converting down to a finer grain of measurement.